Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.8.4 as builder
+RUN go get -u github.com/sparrc/gdm
+COPY Godeps /go/src/github.com/influxdata/influxdb/Godeps
+WORKDIR /go/src/github.com/influxdata/influxdb
+RUN gdm restore
+COPY . /go/src/github.com/influxdata/influxdb
+RUN go install ./cmd/...
+
+FROM debian:stretch
+COPY --from=builder /go/bin/* /usr/bin/
+RUN mkdir -p /etc/influxdb && \
+    influxd config > /etc/influxdb/influxdb.conf
```

Comment:
This shouldn't be `influxd config`, should copy in the the `/etc/config.sample.toml` as the config.

Reply:
Done.